tools/lib/css_parser.py

```diff
@@ -61,6 +61,133 @@ def is_fluff_token(token: Token) -> bool:
 
     return i, text
 
+#### Begin CSS classes here
```

Comment:
This is the class reordering issue.

Reply:
What's the class reordering issue?

Reply:
Several annotations of the subsequent classes require `CssSelector` to be defined first, and so I had to reorder `CssSelector` to be defined first.

Reply:
The other solution is to use forward reference. I think I will just do that to reduce the loc of the patch.